internal/github/checkStalePRs.go

```diff
@@ -0,0 +1,71 @@
+package github
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"strings"
+	"time"
+
+	"github.com/google/go-github/v60/github"
+
+	"github.com/chia-network/github-bot/internal/config"
+)
+
+// CheckStalePRs will return a list of PR URLs that have not been updated in the last 7 days by internal team members.
+func CheckStalePRs(githubClient *github.Client, internalTeam string, cfg config.LabelConfig) ([]string, error) {
+	var stalePRUrls []string
+	cutoffDate := time.Now().AddDate(0, 0, -7) // 7 days ago
+	teamMembers, err := GetTeamMemberList(githubClient, internalTeam)
+	if err != nil {
+		return nil, err
+	}
+
+	for _, fullRepo := range cfg.LabelCheckRepos {
+		log.Println("Checking repository:", fullRepo.Name)
+		parts := strings.Split(fullRepo.Name, "/")
+		if len(parts) != 2 {
+			return nil, fmt.Errorf("invalid repository name - must contain owner and repository: %s", fullRepo.Name)
+		}
+		owner, repo := parts[0], parts[1]
```

Comment:
Sort of a nitpick but duplicate logic could just be its own function

Reply:
Same logic seen in [internal/github/checkPendingCI.go](https://github.com/Chia-Network/github-bot/pull/9/files#diff-9c30dc50701d173ae90de49d485d9031faf7087a7b7359705cc4dee52bea329a)

Reply:
I thought of this while adding the code here and thought that this might come up. I will go ahead and make this a helper function.

Reply:
I did integrate this the helper. But, I also needed that logic in checkPendingCI.go because I need the owner repo variables.